### ISSUE TITLE
streamtape.com

### ISSUE THREAD

**Issue by ghajini** [uBlockOrigin/uAssets#7308]
### URL(s) where the issue occurs

`https://streamtape.com/v/Yy3lyObWZ3UvDxy/The.Photograph.2020.720p.WEBRip.x264.AAC-%5BYTS.MX%5D.mp4`

### Describe the issue

Unblocked Popup ads

### Screenshot(s)

https://user-images.githubusercontent.com/20338483/80924421-6a8e0000-8da6-11ea-80d7-49584c29378c.jpg)

### Versions

- Browser/version: firefox a.
- uBlock Origin version: 1.26.2

### Settings

-  uBO's default settings

**Comment by ghajini** [comment]
`https://streamtape.com/v/29z7WXO663uZL7R/The.Personal.History.of.David.Copperfield.2020.720p.WEBRip.800MB.x264-GalaxyRG.mkv`

New ad

`||ptowhafuzi.pro^`

**Comment by ghajini** [comment]
NSFW

`https://streamtape.net/v/1B0YzbXJxKceklm/EBCSKMHD_%282020%29_www.SkymoviesHD.Me_720p_HEVC_UnRated_HDRip_Hindi_S01E01_x265_AAC.mkv`

Popup ads ,redirections on reaching site continuously

Firefox 80 , opera browser stable
Ublock origin 1.19.2

**Comment by ghajini** [comment]
Right click is disabled ,update existing rule in uBlock annoyances

**Comment by ghajini** [comment]
### URL=

`https://streamtape.com/v/rkMorDlA34hbzL6/DLDYHSKMHD_%282006%29_Www.SkymoviesHD.nl_720p_HEVC_HDRip_Hindi_x265.mkv`

### PLATFORM= OPERA Browser, ublock origin 1.30.9b0

In FIREFOX it work

### Issue=
download doesn't work
download request is quickly closed by easylist filter `*$popup,3p,domain=streamtape.com`

![image](https://user-images.githubusercontent.com/20338483/97671195-c7e0e080-1aad-11eb-9106-a9bb567626ee.png)

**Comment by mapx-** [comment]
@ryanbr

**Comment by ghajini** [comment]
in firefox works,  easylist filter is not triggered?

**Comment by mapx-** [comment]
yes, there are internal differences on how FF / chromium browsers manage the popups

**Comment by lain566** [comment]
@ghajini How about
```
streamtape.com##+js(ra, target, #downloadvideo)
streamtape.com##+js(set, towait, 0)
```

**Comment by ghajini** [comment]
1.
@lain566 works ,make them (`streamtape.*`)

2.
also this filter 
`streamtape.*##+js(acis, Math, XMLHttpRequest)` found in ublock filters 
 doesn't work at `https://streamtape.com/e/61Yb1dQVzof92vz/Table.for.Five.1983.720p.BluRay.H264.AAC-RARBG.mp4`

but work at
 `https://streamtape.com/v/61Yb1dQVzof92vz/Table.for.Five.1983.720p.BluRay.H264.AAC-RARBG.mp4`

**Comment by ghajini** [comment]
Mirror of `streamtape.com`

###URL=
`https://strtape.tech/e/L1WkYPwAXOTbXa`
link came from `https://ww12.movisubmalay.surf/kamen-rider-zi-o-the-movie-over-quartzer-2019/`

### Issue=
Antiadblock,popups

![image](https://user-images.githubusercontent.com/20338483/107525058-75e74500-6bdc-11eb-9e58-653fdd48fb4c.png)

**Comment by ghajini** [comment]
`https://ww12.movisubmalay.surf/tenet-2020/`

`server 5` which is streamtape still gives antiadblock

but when visited individually to a site `https://strtape.tech/e/e2jQz7KxOQcYD2K` antiadblock isn't triggered

**Comment by ghajini** [comment]
video is hidden if adblocker active

`https://streamtape.com/v/BQzklvDjrOsypPo/`

### ubo on=
![image](https://user-images.githubusercontent.com/20338483/109858376-be48ce80-7c81-11eb-869a-b6567d777fd8.png)

### ubo off=
![image](https://user-images.githubusercontent.com/20338483/109858761-3a431680-7c82-11eb-80ae-d7b6f3fa437e.png)

**Comment by ghajini** [comment]
Popup Ads
(new variant of site streamtape)

***URL=***

`https://strcloud.in/v/A0jRRKRgxgHmxo`

source link from `series8.me`

**Firefox nightly A with ublock origin default settings**

**Comment by ghajini** [comment]
`https://streamta.site/e/W14BZ7RP6pibbmV/Homeland.S08E04.1080p.AMZN.WEB-DL.x265.mkv`

`https://stape.fun/e/W14BZ7RP6pibbmV/Homeland.S08E04.1080p.AMZN.WEB-DL.x265.mkv`

popup ads

**Comment by Yuki2718** [comment]
> 
> `https://stape.fun/e/W14BZ7RP6pibbmV/Homeland.S08E04.1080p.AMZN.WEB-DL.x265.mkv`
> 

I can't reproduce.

**Comment by partingscientist** [comment]
I hope commenting here is okay instead of opening a new issue.

Streamtape triggers a debugger statement when opening developer console.
Here's the link: `https://streamtape.com/v/kkbyVLePjWCO1VY/hl_edit.mp4`

<details>

<summary>Here's my configuration</summary>

```yaml
uBlock Origin: 1.48.5b1
Firefox: 111
filterset (summary): 
  network: 198606
  cosmetic: 200834
  scriptlet: 39242
  html: 1094
listset (total-discarded, last-updated): 
  added: 
    https://gitlab.com/magnolia1234/bypass-paywalls-clean-filters/-/raw/main/bpc-paywall-filter.txt: 923-0, 20h.7m
    https://raw.githubusercontent.com/DandelionSprout/adfilt/master/ClearURLs%20for%20uBo/clear_urls_uboified.txt: 700-8, 20h.5m
    https://raw.githubusercontent.com/DandelionSprout/adfilt/master/LegitimateURLShortener.txt: 2147-77, 20h.6m
    adguard-generic: 69293-5321, 3d.20h.30m
    adguard-spyware-url: 962-165, 3d.20h.27m
    block-lan: 46-0, 4d.23h.24m
    curben-phishing: 63256-29, 20h.4m
    curben-pup: 192-0, 20h.10m
    adguard-annoyance: 65836-785, 3d.20h.28m
    fanboy-annoyance: 81594-5686, 3d.18h.23m
    ublock-annoyances: 4854-4, 3d.20h.25m
    dpollock-0: 11349-470, 1d.3h.30m
    IDN-0: 5473-1, 3d.20h.26m
  default: 
    user-filters: 47-0, never
    ublock-filters: 33991-76, 3d.20h.29m
    ublock-badware: 5481-0, 3d.20h.24m
    ublock-privacy: 344-3, 9d.15h.20m
    ublock-abuse: 78-0, 9d.15h.19m
    ublock-unbreak: 1955-2, 3d.20h.31m
    ublock-quick-fixes: 382-3, 20h.9m
    easylist: 65103-777, 2d.2h.16m
    easyprivacy: 30767-1859, 1d.22h.15m
    urlhaus-1: 6870-0, 20h.8m
    plowe-0: 3751-2, 8d.19h.30m
filterset (user): [array of 46 redacted]
hostRuleset: 
  added: [array of 83 redacted]
modifiedUserSettings: 
  advancedUserEnabled: true
  colorBlindFriendly: true
  contextMenuEnabled: false
modifiedHiddenSettings: 
  userResourcesLocation: [redacted]
supportStats: 
  allReadyAfter: 1317 ms (selfie)
  maxAssetCacheWait: 217 ms
popupPanel: 
  blocked: 13
  blockedDetails: 
    streamtape.com: 2
    acscdn.com: 2
    adforcast.com: 3
    gstatic.com: 1
    ijorecepous.com: 1
    yandex.ru: 1
    zimpolo.com: 3
```
</details>

Tested on a clean profile with default configuration, but I'm reporting from my dirty profile for convenience, hence the attached configuration.

**Comment by mapx-** [comment]
add

`streamtape.com##^script:has-text(/0x|\\x/)`
